src/pages/trade/ui/summary.tsx

```diff
@@ -68,57 +60,82 @@ export const Summary = () => {
   return (
     <div className='flex flex-wrap items-center gap-x-4 gap-y-2'>
       <SummaryCard title='Price' loading={isLoading}>
-        {data && (
+        <Text detail color='text.primary'>
+          {data && 'price' in data ? roundTrim(data.price) : '-'}
+        </Text>
+      </SummaryCard>
+      <SummaryCard title='24h Change' loading={isLoading}>
+        {data && 'noData' in data && (
           <Text detail color='text.primary'>
-            {round(data.price, 6)}
+            -
           </Text>
```

Comment:
If no summary is available defaulting to dashes for the value instead of the error message